picard/formats/id3.py

```diff
@@ -494,6 +494,19 @@ def _get_tags(self, filename):
         except mutagen.id3.ID3NoHeaderError:
             return compatid3.CompatID3()
 
+    def _load_preserved_config(self, metadata):
+        # Adding additional config to metadata to prevent unecessary saving
+        metadata['~config:rating_steps'] = config.setting['rating_steps']
+        metadata['~config:rating_user_email'] = config.setting['rating_user_email']
```

Comment:
Adding ~ metadata is not in itself a privacy issue as ~ data is not (with some exceptions) saved to media files.

However, email addresses are saved to the id3 rating tag - this is how the rating tag is defined in the id3 standard, not a choice made by Picard.

BUT, I council against storing config or indeed non-track related data in ~ metadata. Just because it is not visible in the standard UI, doesn't make this a good idea.